sdk/python/kfp/registry/registry_client.py

```diff
@@ -64,29 +70,28 @@ def __call__(self,
 class RegistryClient:
     """Registry Client class for communicating with registry hosts."""
 
-    def __init__(
-        self,
-        host: str,
-        auth: Optional[Union[requests.auth.AuthBase,
-                             credentials.Credentials]] = None
-    ) -> None:
+    def __init__(self,
+                 host: Optional[str],
+                 auth: Optional[Union[requests.auth.AuthBase,
+                                      credentials.Credentials]] = None,
+                 config_file: Optional[str] = None,
+                 auth_file: Optional[str] = None) -> None:
         """Initializes the RegistryClient.
 
         Args:
-            host: The address of the registry host.
-            auth: Authentication using python requests or google.auth credentials.
+            host: The address of the registry host. The host needs to be specified here
+                or in the config file.
+            auth: Optional. Authentication using python requests or google.auth credentials.
+            config_file: Optional. The location of the local config file. If not specified,
+                defaults to LOCAL_REGISTRY_CONTEXT (if it exists).
+            auth_file: Optional. The location of the local config file that contains the
+                authentication token. If not specified, defaults to
+                LOCAL_REGISTRY_CREDENTIAL (if it exists).
         """
-        self._host = host.rstrip('/')
+        self._host = ''
```

Comment:
There are 3 places where `self._host` could come from: the argument during initialization of `RegistryClient`, a default config file, or a user-specified config file.. I think it'd be good to just gather all that logic within `.load_config()` so that it's more concise, WDYT?

Reply:
That sounds good to me 👍🏻